model_analyzer/config/generate/optuna_run_config_generator.py

```diff
@@ -366,38 +393,44 @@ def _create_trial_objective(
         return objective
 
     def _get_objective_concurrency(self, trial_objectives: TrialObjectives) -> int:
+        max_batch_size = trial_objectives.get("max_batch_size", 1)
```

Comment:
Any reason why only one of these was pulled out? Seems there was an assumption that both existed before. Now we are keeping the assumption for one of them?

Reply:
A model doesn't have to support batching (in which case we want to use 1 for the formula), but it will always have an instance count.